src/styles.scss

```diff
@@ -51,3 +51,22 @@ body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+.mat-snack-bar-container{
+  color: white;
+  font-weight: 500;
+  &.success{
+    background-color: #28a745!important;
```

Comment:
Also we have file `colors.scss` where app colors are defined. You should reuse those that are defined or /and add missing ones and import them when necessary.